packages/block-library/src/paragraph/edit.js

```diff
@@ -170,7 +182,7 @@ function ParagraphBlock( {
 						: __( 'Block: Paragraph' )
 				}
 				data-empty={ RichText.isEmpty( content ) }
-				placeholder={ placeholder || __( 'Type / to choose a block' ) }
+				placeholder={ placeholder }
 				data-custom-placeholder={ placeholder ? true : undefined }
```

Comment:
I think the change breaks `data-custom-placeholder` (I have no idea what it does, but I know the placeholder text is filterable, so I guess it's related).

Previously it would be `undefined` when the default 'Type / to choose a block' text is used for the placeholder.

Reply:
Oh it was right there :D what is that ....